### ISSUE TITLE
Changes to button inputs

### ISSUE THREAD

**Issue by s-duffey** [gctools-outilsgc/design-system-code#91]
Button inputs page needs updates for accessibility:

- [x] Review component examples (switch esp.)
- [x] Checkbox examples
- [x] Badges need black text (#96A8B2)
